packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.Basic.Example.tsx

```diff
@@ -1,79 +1,103 @@
 import * as React from 'react';
-import { ContextualMenuItemType } from 'office-ui-fabric-react/lib/ContextualMenu';
-import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
-import './ContextualMenuExample.scss';
+import { ContextualMenu, ContextualMenuItemType, IContextualMenuItem } from 'office-ui-fabric-react/lib/ContextualMenu';
+
+export interface IContextualMenuBasicExampleState {
+  showContextualMenu?: boolean;
+}
+const menuItems: IContextualMenuItem[] = [
+  {
+    key: 'newItem',
+    text: 'New',
+    onClick: () => console.log('New clicked')
+  },
+  {
+    key: 'divider_1',
+    itemType: ContextualMenuItemType.Divider
+  },
+  {
+    key: 'rename',
+    text: 'Rename',
+    onClick: () => console.log('Rename clicked')
+  },
+  {
+    key: 'edit',
+    text: 'Edit',
+    onClick: () => console.log('Edit clicked')
+  },
+  {
+    key: 'properties',
+    text: 'Properties',
+    onClick: () => console.log('Properties clicked')
+  },
+  {
+    key: 'linkNoTarget',
+    text: 'Link same window',
+    href: 'http://bing.com'
+  },
+  {
+    key: 'linkWithTarget',
+    text: 'Link new window',
+    href: 'http://bing.com',
+    target: '_blank'
+  },
+  {
+    key: 'linkWithOnClick',
+    name: 'Link click',
+    href: 'http://bing.com',
+    onClick: (ev: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>) => {
+      alert('Link clicked');
+      ev.preventDefault();
+    },
+    target: '_blank'
+  },
+  {
+    key: 'disabled',
+    text: 'Disabled item',
+    disabled: true,
+    onClick: () => console.error('Disabled item should not be clickable.')
+  }
+];
+
+export class ContextualMenuBasicExample extends React.Component<{}, IContextualMenuBasicExampleState> {
```

Comment:
Another thing you can take or leave, I've been trying to convert the examples to function components since that's the pattern we want to encourage now. So if you don't mind converting these ones, that would be nice (but not essential).